modules/widgets/simple-payments.php

```diff
@@ -56,8 +56,11 @@ function __construct() {
 				)
 			);
 
-			if ( is_customize_preview() ) {
```

Comment:
Why did we move this check further down? Do we need to enqueue these styles for each section, or could we limit that just to customizer and posts/pages?

Reply:
Indeed we do need the styles for the `widgets.php` page as well.
Although, you're right, we shouldn't show it everywhere, but only in Customizer and Widgets.

Reply:
See c4a85f2